config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
is this a regex-type thing? ie. will `passw` cause `password`/`password_confirmation` to be filtered out?

Reply:
I assume so. This is just the new Rails default provided by `app:update`.

Reply:
According to https://guides.rubyonrails.org/configuring.html

>Parameters filter works by partial matching regular expression.

Reply:
ok. thnx